### PR TITLE
Implemented Fast Fourier Transform

### PR DESCRIPTION
**Pull request by Aryan-d13**
## 🛠️ Closes Issue #1313 
Issue no #1313

Implemented Fast Fourier Transform in Java, C, C++, Python

- [✔️ ] My code follows the code style of this project.
- [ ✔️] This PR does not contain plagiarized content.
- [ ✔️] The title of my pull request is a short description of the requested changes.

## 📄 Note to reviewer
I've Implemented the FFT and tested the Output in the most Effiecient way I could, I had to create the Complex class in Java on my own as it wasn't available on Apache servers anymore

📷 Screenshots
image
image
image
image

### CODE REVIEW

**Comment by Aryan-d13** [comment]
Closes Issue #1313

**Review by Kumar-laxmi — CHANGES_REQUESTED** [review]
You are required to put the screen shot of your output and not the code, code is already visible in the CodeSpace

File: Java/Signal Processing/FFT.java
Comment (zugzwang03) [inline Java/Signal Processing/FFT.java:1]: change this name here

**Comment by Aryan-d13** [comment]
@Kumar-laxmi @zugzwang03 Please review the changes

**Review by 072arushi — APPROVED** [review]

**Pull request closed without merging**